english/dig/cup-cocoa/cup-cocoa.md

```diff
@@ -2,8 +2,17 @@ You enjoy the warm cup of cocoa and you have a conversation with the big bird th
 Big Bird from Sesame Street. You ask him what he is doing in the forest and ask him if he would like
 a cup of "cocoa". He starts to correct you and says it is pronounced "co-co-aaaaaaaaa" and not "co-co".
 This pointless debate ensues for a few minutes until he concedes that it is indeed cocoa after you google
-the correct pronunciation. Then you ask him again what he is doing in the forest. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.